release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -148,6 +148,46 @@ int result = -9223372036854775808; // error: '9223372036854775808' is out of ran
   - `createEnumDeclarationNode`
   - `createClassDefinitionNode`
 
+- Restricted query expression outer join type-binding to be inferred as a nillable type 
+    Earlier, the type binding for outer join could be any type-descriptor but now its restricted to be 'var' so that it will be inferred as a nillable type.
```

Comment:
```suggestion
    Earlier, the type binding for outer join could be any type-descriptor but now it's restricted to be 'var' so that it will be inferred as a nillable type.
```